errors on build in vs2010

#7145  
Version: 3.1.0-1195-gbf4c5be

Error:
"C:\Lib\opencv\build_x86_pre\OpenCV.sln" (default target) (1) ->
"C:\Lib\opencv\build_x86_pre\ALL_BUILD.vcxproj.metaproj" (default target) (2) -
>
"C:\Lib\opencv\build_x86_pre\apps\interactive-calibration\opencv_interactive-ca
libration.vcxproj.metaproj" (default target) (62) ->
"C:\Lib\opencv\build_x86_pre\apps\interactive-calibration\opencv_interactive-ca
libration.vcxproj" (default target) (63) ->
(ClCompile target) ->
  ..\..\..\sources\opencv\apps\interactive-calibration\frameProcessor.cpp(256):
 error C2668: 'std::pow' : ambiguous call to overloaded function [C:\Lib\opencv
\build_x86_pre\apps\interactive-calibration\opencv_interactive-calibration.vcxp
roj]
  ..\..\..\sources\opencv\apps\interactive-calibration\frameProcessor.cpp(257):
 error C2668: 'std::pow' : ambiguous call to overloaded function [C:\Lib\opencv
\build_x86_pre\apps\interactive-calibration\opencv_interactive-calibration.vcxp
roj]

--- a/apps/interactive-calibration/frameProcessor.cpp
+++ b/apps/interactive-calibration/frameProcessor.cpp
@@ -253,8 +253,8 @@ CalibProcessor::CalibProcessor(cv::Ptr<calibrationData> data, captureParameters 
     mCapuredFrames = 0;
     mNeededFramesNum = capParams.calibrationStep;
     mDelayBetweenCaptures = static_cast<int>(capParams.captureDelay * capParams.fps);
-    mMaxTemplateOffset = std::sqrt(std::pow(mCalibData->imageSize.height, 2) +
-                                   std::pow(mCalibData->imageSize.width, 2)) / 20.0;
+    mMaxTemplateOffset = std::sqrt(static_cast<float>(mCalibData->imageSize.height * mCalibData->imageSize.height) +
+                                   static_cast<float>(mCalibData->imageSize.width * mCalibData->imageSize.width)) / 20.0;
     mSquareSize = capParams.squareSize;
     mTemplDist = capParams.templDst;
 

